extractor/extractor.py

```diff
@@ -0,0 +1,223 @@
+import bs4
```

Comment:
Instead of different functions, create a class Parser/Extractor, and convert these functions to class methods.